CMakeLists.txt

```diff
@@ -4,17 +4,16 @@ file(STRINGS "version.txt" __C_BINDINGS_VERSION)
 project(c_bindings VERSION ${__C_BINDINGS_VERSION} LANGUAGES CXX)
 unset(__C_BINDINGS_VERSION)
 
-list (APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
-
 include(CTest)
 
 set(REQUIRED_BOOST_VERSION 1.58)
 find_package(Boost ${REQUIRED_BOOST_VERSION} REQUIRED)
 
-include(bindings)
+# if used via FetchContent/add_subdirectory() we need to make the add_bindings_library() available here
+include(${CMAKE_CURRENT_LIST_DIR}/cmake/bindings.cmake)
```

Comment:
I don't understand the comment. `add_bindings_library` also needs to be made available in case we compile it standalone?

Reply:
No, because standalone doesn't have to compile anything (if tests are disabled).